Project.toml

```diff
@@ -9,21 +9,23 @@ DataStructures = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
 JSON = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
 JSON3 = "0f8b85d8-7281-11e9-16c2-39a750bddbf1"
 LightGraphs = "093fc24a-ae57-5d10-9952-331d41423f4d"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 PyCall = "438e738f-606a-5dbb-bf0a-cddfbfd45ab0"
 PyPlot = "d330b81b-6aea-500a-939a-2ce795aea3ee"
```

Comment:
similarly here - do we really need all these packages as dependencies?

Reply:
@aleant93 you need to clean up conflicts. Especially, I have made a package version cleanup and in your code old versions show up. For example we do not use `JSON.jl` at all and I have removed this dependency.

Reply:
My fault! When I first created this branch, my local master was up-to-date with remote. Yesterday, I did not check again before doing this PR. Working on that!